distribution/ddf-common/src/main/resources/security/configurations.policy

```diff
@@ -0,0 +1,33 @@
+priority "grant";
```

Comment:
Should this file be called something like cdm.policy

Reply:
I will update this with a comment indicating its intended usage, but after talking with @stustison , it was decided that there will be one policy file for any permissions that are configuration related, and that this file will contain any necessary sections.